lib/elixir/lib/path.ex

```diff
@@ -3,21 +3,25 @@ defmodule Path do
   This module provides conveniences for manipulating or
   retrieving file system paths.
 
-  The functions in this module may receive a chardata as
-  argument (i.e. a string or a list of characters / string)
-  and will always return a string (encoded in UTF-8). If a binary
+  The functions in this module may receive a `t:IO.chardata/0` as
```

Comment:
Linking to the type is good but I am afraid it will make it less clearer. What if we keep the regular name and then say “see t:IO.chardata”?

Reply:
Sure, will do that and merge 👍